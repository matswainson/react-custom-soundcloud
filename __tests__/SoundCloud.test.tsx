import * as React from 'react';
import SoundCloud from '../src';
import { create, ReactTestRenderer } from 'react-test-renderer';

let soundcloud: ReactTestRenderer;

it('renders correctly', () => {
  // soundcloud = create(<SoundCloud track="370226924" />);
  soundcloud = create(<SoundCloud track="370226924" />);
  soundcloud.toJSON();
  expect(soundcloud).toMatchSnapshot();
});