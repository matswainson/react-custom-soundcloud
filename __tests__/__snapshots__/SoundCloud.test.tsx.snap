// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-player"
>
  <iframe
    allow="autoplay"
    frameBorder="no"
    height="450"
    scrolling="no"
    src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/370226924&amp;auto_play=false&amp;hide_related=false&amp;show_comments=true&amp;show_user=true&amp;show_reposts=false&amp;visual=true"
    title="Soundcloud"
    width="100%"
  />
</div>
`;
